File: rizum-pt-to-ps-bridge/sp_plugin/rizum_sp_to_ps/metadata.py
"""Metadata constants and sidecar helpers shared by Painter code."""

UID_SUFFIX_PREFIX = "\u2021"
SIDECAR_EXTENSION = ".rizum.json"


def format_uid_suffix(uid_hex):
    """Format the stable Photoshop layer-name suffix for a Painter uid."""
    return f"{UID_SUFFIX_PREFIX}{uid_hex}"

File: rizum-pt-to-ps-bridge/ps_plugin/src/metadata.js
"use strict";

const UID_SUFFIX_PREFIX = "\u2021";
const SIDECAR_EXTENSION = ".rizum.json";

function formatUidSuffix(uidHex) {
  return `${UID_SUFFIX_PREFIX}${uidHex}`;
}

module.exports = {
  UID_SUFFIX_PREFIX,
  SIDECAR_EXTENSION,
  formatUidSuffix
};
